### ISSUE TITLE
Basic Neural Network

### ISSUE THREAD

**Issue by ccwang002** [11thDeepLearningGroup2015/MLDS_HW1#4]
Implement traditional NN

- [ ] softmax neuron
- [ ] gradient decent solver w. back propagation
- [ ] multiple (3-4) layers
- [ ] weight initialization by randomization w. variance control (see Stanford CNN [course note])

[course note]: http://cs231n.github.io/neural-networks-2/

**Comment by wush978** [comment]
You are so strong!
